libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1509,6 +1536,10 @@ string YulUtilFunctions::copyArrayToStorageFunction(ArrayType const& _fromType,
 						let <elementValues> := <readFromCalldataOrMemory>(srcPtr)
 					</fromMemory>
 
+					<?fromStorage>
+						let <elementValues> := srcPtr
+					</fromStorage>
+
```

Comment:
Do we need this, as for value array type we call other function?

Reply:
I don't understand - we need it for non-value types, don't e?

Reply:
Yes we need. But was thinking this PR is only for value types and it is going.